ui/core/components/individual_sim_ui/gem_summary.ts

```diff
@@ -0,0 +1,73 @@
+import { SimUI } from "../../sim_ui";
+import { Component } from "../../components/component";
+import { Player } from "../../player";
+import { setItemQualityCssClass } from "../../css_utils";
+import { ActionId } from "../../proto_utils/action_id";
+import { UIGem as Gem } from '../../proto/ui.js';
+import { ContentBlock } from "../content_block";
+
+interface GemSummaryData {
+	gem: Gem
+	count: number
+}
+
+export class GemSummary extends Component {
+	private readonly simUI: SimUI;
+	private readonly player: Player<any>;
+
+	private readonly container: ContentBlock;
+
+	constructor(parent: HTMLElement, simUI: SimUI, player: Player<any>) {
+		super(parent, 'gem-summary-root');
+		this.simUI = simUI;
+		this.player = player;
+
+    this.container = new ContentBlock(this.rootElem, 'gem-summary-container', {
+      header: {title: 'Gem Summary'}
+    });
+
+    player.gearChangeEmitter.on(() => this.updateTable());
+	}
```

Comment:
Minor nit, but looks like indentation got messed up here maybe?

Reply:
Ah looks like my editor made these new lines use spaces instead of tabs. Nice catch! It looked fine for me because I use 2-space tab sizing but I'll bet it looks funky if you use anything else